test/benchmarks/material/button-harness/protractor.perf-spec.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {HarnessLoader} from '@angular/cdk/testing';
+import {MatButtonHarness} from '@angular/material/button/testing/button-harness';
+import {runBenchmark} from '@angular/dev-infra-private/benchmark/driver-utilities';
+import {ProtractorHarnessEnvironment} from '@angular/cdk/testing/protractor';
+
+let loader: HarnessLoader;
+
+describe('perf test for basic protractor harness', () => {
+	it('should load the protractor harness environment', async () => {
+		await runBenchmark({
+			id: 'initial-harness-load',
+			url: '',
+			ignoreBrowserSynchronization: true,
+			params: [],
+			work: () => { loader = ProtractorHarnessEnvironment.loader(); },
+		});
+	});
+
+	it('should retrieve all of the buttons', async () => {
```

Comment:
For comparison, can we add versions of these tests that do the same thing but without harnesses?